main/domoticz.cpp

```diff
@@ -171,6 +172,7 @@ bool g_bStopApplication = false;
 bool g_bUseSyslog = false;
 bool g_bRunAsDaemon = false;
 bool g_bDontCacheWWW = false;
+_eWebCompressionMode g_wwwCompressMode;
```

Comment:
Could you set the default here (http::server::WWW_USE_GZIP)
Then the 'else' if (cmdLine.HasSwitch("-wwwcompress")) here is not needed

Reply:
Aha! ***That*** is why I had it initialized to 0 when it was still an int...

Reply:
Yes, that's why it is good everything is triple checked ;)